app/browser/api/ledger.js

```diff
@@ -2321,7 +2321,7 @@ const run = (state, delayTime) => {
     })
   }
 
-  if (typeof delayTime === 'undefined' || !client) {
+  if (typeof state === 'undefined' || typeof delayTime === 'undefined' || !client) {
```

Comment:
Can we do `state == null`? This way we prevent null pass as well?

Reply:
sure- that works for me!